include/dos_inc.h

```diff
@@ -87,6 +87,12 @@ enum class DosReturnMode : uint8_t {
 	TerminateAndStayResident = 3
 };
 
+enum class DiskType : uint8_t {
+	Floppy   = 0,
+	HardDisk = 1,
+	CDRom    = 2,
+};
```

Comment:
Just `enum class DiskType { Floppy,  HardDisk, CdRom };` (and `CdRom`, not `CDRom`)